### DISCUSSION TITLE
yml instead of xml

### DISCUSSION THREAD

**Discussion by Drsheppard01** [labwc/labwc#2472] [feature-requests]
Obviously labwc has become not just a port of openbox to wayland, but a project in its own right. I think the use of an outdated configuration architecture (rc.xml, autorun, environment, menu.xml) makes it difficult for new users to use.

I suggest using a more user friendly yml format and not separating all configurations

**Comment by johanmalm** [comment]
See:

- https://github.com/labwc/labwc/pull/2106
- https://github.com/johanmalm/yaml2xml/
